sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/implementation/ManagementChannelTests.java

```diff
@@ -524,6 +525,30 @@ void cancelScheduledMessagesWithEmptyIterable() {
                 .verifyComplete();
     }
 
+    @Test
+    void createRule() {
+        // Arrange, act, assert
+        final String ruleName = "foo-bar";
+        CreateRuleOptions options = new CreateRuleOptions();
+        StepVerifier.create(managementChannel.createRule(ruleName, options))
+            .verifyComplete();
+    }
+
+    @Test
+    void getRules() {
+        // Arrange, act, assert
+        StepVerifier.create(managementChannel.getRules())
```

Comment:
It would be worthwhile to create a message _with_ some rules so you can check it deserialized properly.